kustomize/components/route/route.varreference.yaml

```diff
@@ -0,0 +1,3 @@
+varReference:
```

Comment:
Adding a custom `varReference` following [this inspiration](https://github.com/kubernetes-sigs/kustomize/issues/1938#issuecomment-567974171).